src/Microsoft.DotNet.Arcade.Sdk/tools/VSTest.targets

```
@@ -39,6 +39,12 @@
     <MakeDir Directories="@(_OutputFiles->'%(RootDir)%(Directory)')"/>
     <Delete Files="@(_OutputFiles)" />
 
+    <!--
+      Add command line to the log.
+    -->
+    <Exec Command="echo === COMMAND LINE === >> %(TestToRun.ResultsStdOutPath)" />
+    <Exec Command="echo $(_TestRunnerCommand) >> %(TestToRun.ResultsStdOutPath)" />
```

```suggestion
    <Exec Command="echo === COMMAND LINE === >> %(TestToRun.ResultsStdOutPath)
                                   echo $(_TestRunnerCommand) >> %(TestToRun.ResultsStdOutPath)" />
```
has the same effect, of course.